ee/clickhouse/queries/test/test_paths.py

```diff
@@ -1720,6 +1720,7 @@ def test_event_inclusion_exclusion_filters(self):
         )
 
     @snapshot_clickhouse_queries
+    @freeze_time("2012-01-01 03:20")
```

Comment:
Needs to be greater than timestamps in the test for tests to pass I think

```suggestion
    @freeze_time("2012-01-11 03:20")
```